vuepress/docs/next/docs/curate/bundle-comparison.md

```diff
@@ -0,0 +1,39 @@
+
+Some of the major improvements with the docker-based bundles are:
+* Simplified bundle directory structure
+* Just one source git repository required with Docker image repositories created automatically as needed
+* Optimized file storage and transfer using a layered file-system, beneficial for large projects
+* Standardized bundle address format
+* A decoupled build and deployment process which is easier to understand, modify, and troubleshoot 
+
+For specifications on docker-based bundles, see the [Bundle Details](ecr-bundle-details.md) page. 
+### Bundle Comparison Table
+|Attribute | [git-based bundles](../../../v7.0/docs/curate/ecr-bundle-details.md)| [docker-based bundles](ecr-bundle-details.md)|
+| :- | :-------------------- | :---------------------  |
+|`descriptorVersion`| v1| v5
+| Bundle specifications |Defined with descriptor files | Built with [ent bundle CLI](../getting-started/ent-bundle.md) tool
+| Versioning |Bundle version set by git tags |Bundle version set by Docker Tags. For MFEs, the version defaults to 1.0.0 unless set in the [entando.json](ecr-bundle-details.md). 
+|Plugin Ingress| Generated by Entando or set in the plugin descriptor.yaml | Generated by Entando or customized in the [bundle descriptor](ecr-bundle-details.md#entando-bundle-conventions), entando.json 
+|Identifiers| Designated Bundle ID and Plugin ID | Bundles, MFEs, & Plugins have unique global IDs|
+|Initialize from Entando Hub| As git clone  |[Direct initialization from Hub](../../tutorials/create/mfe/react.md)(TODO-switch to appropriate tutorial)  with `ent init BUNDLE-NAME –from-hub`|
+|Bundle Creation | `ent prj` commands| New modular `ent bundle` commands
+|Deploy & Install| `ent ecr deploy` and `ent ecr install` | No change 
+|Export Bundle|`ent bundler`| Not Available 
+|JHipster Integrated Blueprint Project| `ent jhipster blueprint` creates [MFE & MS template project](../../tutorials/create/ms/generate-microservices-and-micro-frontends.md)| [Convert Blueprint project to v5 bundle](../../tutorials/create/ms/generate-microservices-and-micro-frontends.md) (TODO-change link to new tutorial)
+|APIs| Not Available | [Manage APIs with `ent bundle` commands](../getting-started/ent-api.md). MFEs can make API claims to MSs within the namespace
+||| MFEs support service discovery by using the property apiClaim
+|Local Services |Not Available | Default services now include PostgreSQL, MySQL, and Keycloak. Additional services can be added with the [`ent bundle svc` commands](../getting-started/ent-svc.md).
+|MFE custom UI| Add with manual scripts| Custom UI is auto-generated in FTL format
+|Data transfer | Manual | New ways to pass values or information to widgets, bundles, or expose in the APIs by [utilizing context parameters, system parameters and the ext variable](ecr-bundle-details.md).
```

Comment:
sentence fragment is awkward - "or expose in the APIs..." what do you mean by this?

Reply:
the verb changed so the subject needs to be reintroduced somehow... "New ways to pass values to widgets and bundles or expose information in APIs by..."